realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -29,18 +27,81 @@
 public class AllTypesRealmProxy extends AllTypes
         implements RealmObjectProxy {
 
-    private static long INDEX_COLUMNSTRING;
-    private static long INDEX_COLUMNLONG;
-    private static long INDEX_COLUMNFLOAT;
-    private static long INDEX_COLUMNDOUBLE;
-    private static long INDEX_COLUMNBOOLEAN;
-    private static long INDEX_COLUMNDATE;
-    private static long INDEX_COLUMNBINARY;
-    private static long INDEX_COLUMNOBJECT;
-    private static long INDEX_COLUMNREALMLIST;
+    static final class AllTypesColumnInfo extends ColumnInfo {
+
+        public final long columnStringIndex;
+        public final long columnLongIndex;
+        public final long columnFloatIndex;
+        public final long columnDoubleIndex;
+        public final long columnBooleanIndex;
+        public final long columnDateIndex;
+        public final long columnBinaryIndex;
+        public final long columnObjectIndex;
+        public final long columnRealmListIndex;
+
+        AllTypesColumnInfo(Table table, ImplicitTransaction transaction) {
```

Comment:
Why pass the transaction as a parameter? you only need the the transaction.getPath() right? So pass that instead.